packages/issuer-sdk-js/src/server/credentials-builder.test.ts

```diff
@@ -42,6 +48,13 @@ describe("buildVerifiableCredentials", () => {
       },
     );
 
+    expect(data.credentialSubject.residentialAddressStreet).toBe("Main St");
+    expect(data.credentialSubject.residentialAddressHouseNumber).toBe("123");
+    expect(data.credentialSubject.residentialAddressAdditionalAddressInfo).toBe("Apt 1");
+    expect(data.credentialSubject.residentialAddressCity).toBe("New York");
+    expect(data.credentialSubject.residentialAddressPostalCode).toBe("10001");
+    expect(data.credentialSubject.residentialAddressCountry).toBe("US");
+
```

Comment:
Can you add a call to `validate`, just to be sure it survives the roundtrip?

Reply:
sure